.github/actions/go-cache/action.yml

```diff
@@ -24,7 +27,7 @@ runs:
           ~/.cache/golangci-lint
           ~/Library/Caches/golangci-lint
           ~\AppData\Local\golangci-lint
-        key: ${{ runner.os }}-go-v2-${{ inputs.key }}-${{ hashFiles('**/go.sum') }}
+        key: ${{ runner.os }}-go-${{ inputs.goversion }}-v4-${{ inputs.key }}-${{ hashFiles('**/go.sum') }}
         restore-keys: |
-          ${{ runner.os }}-go-v4-
-          ${{ runner.os }}-go-v4-${{ inputs.key }}-
+          ${{ runner.os }}-go-${{ inputs.goversion }}-v4-
+          ${{ runner.os }}-go-${{ inputs.goversion }}-v4-${{ inputs.key }}-
```

Comment:
Asi to bude fakt nejake issue s cache.
Pridal som do kluca Go verziu, neda sa to specifikovat nejak pre cely projekt.
Preto novy input.

Prvy krat to preslo, spustam znova a doriesime to v piatok.